src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -235,4 +235,28 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
     isHeavyOrSuper() {
         return this.weightclass === HEAVY_LETTER || this.weightclass === SUPER_LETTER;
     }
+
+    /**
+     * Returns the minimal distance that is required to a previous aircraft before another aircraft is allowed to use the runway.
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method calculateRunwaySeparationDistanceInFeet
+     * @param previousTypeModel {AircraftTypeDefinitionModel} the aircraft type that used the runway before us.
+     * @returns {number} distance in feet
+     */
+    calculateRunwaySeparationDistanceInFeet(previousTypeModel) {
+        if (typeof previousTypeModel.category.srs === 'undefined'
+            || previousTypeModel.category.srs === 3) {
+            return 6000;
```

Comment:
lets set this to a constant to give it a little more context.  **_what_** does the value `6000` mean?

Reply:
done